force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -159,11 +243,12 @@ public with sharing class VariablesDataTypesOperators {
      * @return The number of milliseconds equivalent to the given number of minutes.
      */
     public static Integer convertMinutesToMilliseconds(Integer minutes) {
-        Integer MILLISECONDS_PER_MINUTE = null; // Make this value a constant
-        Integer milliseconds;
-        return null; // Replace null with the variable you used to store the result
+        Integer MILLISECONDS_PER_MINUTE = 60000; // Make this value a constant
+        Integer milliseconds = minutes * MILLISECONDS_PER_MINUTE; // Variable
+        return minutes * milliseconds; // Replace null with the variable you used to store the result
     }    
```

Comment:
Not quite right you are getting the milliseconds per minute on line 247. That is what you want to return. 

But you are multiplying again on line 248